src/NUnitTestAdapter/NUnit/NUnit3FrameworkDriver.cs

```diff
@@ -22,12 +22,8 @@
 // ***********************************************************************
 
 using System;
-using System.Diagnostics;
 using System.IO;
 using System.Collections.Generic;
-using System.Web.UI;
-using System.Xml;
-//using NUnit.Engine.Internal;
 using NUnit.Engine.Extensibility;
 
 namespace NUnit.Engine.Drivers
```

Comment:
This namespace doesn't match folder location.  I think it would be nice if these were matching.